src/components/productView/productView.js

```diff
@@ -84,14 +103,82 @@ const ProductView = ({ t, useRouteDetail: useAliasRouteDetail }) => {
       </ProductViewContext.Provider>
     );
   };
+  */
+  const renderProduct = useCallback(() => {
```

Comment:
required:
confirm this is necessary when compared against the useRouteDetail hook

Reply:
inconclusive, we're going to leave it in for now and reevaluate. it's not hurting us currently. the odd random double/triple load makes it difficult to determine when using the proxy